package.json

```diff
@@ -6,6 +6,19 @@
   "engines": {
     "node": ">=6.13.0"
   },
+  "browserslist": [
+    "> 1%",
+    "ie >= 8",
+    "edge >= 15",
+    "ie_mob >= 10",
+    "ff >= 45",
+    "chrome >= 45",
+    "safari >= 7",
+    "opera >= 23",
+    "ios >= 7",
+    "android >= 4",
+    "bb >= 10"
+  ],
```

Comment:
Neat! I didn't think about this.

Reply:
I had a weird error trying to install semantic that brought me to a rabbit hole of browser compatibility... I left it there because it doesn't seem to hurt.